### PR TITLE
[DEVOPS-1395] build and deploy web vault

### PR DESCRIPTION
**Pull request by michalchecinski**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [x] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Build and deploy the Web Vault to CloudFlare Pages for `EUdevtest` environment.

## Code changes

- **apps/web/config/eudevtest.json:** Add `eudevtest` config for web vault
- **apps/web/package.json** Add build command for `eudevtest` env
- **.github/workflows/build-web.yml** Add build stage in workflow for `eudevtest`
- **.github/workflows/release-eudevtest-web.yml** Add release workflow for `eudevtest`

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: .github/workflows/release-eudevtest-web.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Create GitHub deployment
+        uses: chrnorm/deployment-action@d42cde7132fcec920de534fffc3be83794335c00 # v2.0.5
+        id: deployment
+        with:
+          token: '${{ secrets.GITHUB_TOKEN }}'
+          initial-status: 'in_progress'
+          environment-url: http://web-vault.eudevtest.bitwarden.pw
+          environment: 'Web Vault - eudevtest'
```
[the remark this answers is not in the document]
Comment (michalchecinski) [reply .github/workflows/release-eudevtest-web.yml:19]: I did it myself in Cloudflare, but you can double-check @TJ-tijani 😊

[the remark this answers is not in the document]
Comment (TJ-tijani) [reply .github/workflows/release-eudevtest-web.yml:19]: on the latest request @mimartin12 , are we looking at CF worker environment or just a preview deployment configuration on CF pages for the eudevtest branch?

**Comment by joseph-flinn** [comment]
@michalchecinski Could we actually update https://github.com/bitwarden/clients/blob/master/.github/workflows/deploy-non-prod-web.yml instead of creating a new workflow?

**Comment by michalchecinski** [comment]
@joseph-flinn I've added `eudevtest` as another option alongside `POC2` in `deploy-non-prod-web.yml`. Should I remove the `POC2` one?

File: .github/workflows/release-eudevtest-web.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (michalchecinski) [reply .github/workflows/release-eudevtest-web.yml]: @TJ-tijani could you create one for us? I merged the PR already but I don't have access to the repo settings.

**Pull request merged**